modules/UI/authentication/AuthHandler.js

```diff
@@ -80,8 +81,11 @@ function initJWTTokenListener(room) {
             return;
         }
         if (event.data && event.data.jwtToken) {
-            config.token = event.data.jwtToken;
-            logger.info("Received JWT token:", config.token);
+            const token = event.data.jwtToken;
+
+            APP.store.dispatch(updateConfig({ token }));
```

Comment:
I see why you do this. It's because the code somewhere looks for this in `config.token`, right? I don't think the token is a configuration related entity. Can we move it to the right place in redux and have the web use `APP.store.getState` to read it instead? The more technical debt we can pay the better!